sw.js

```diff
@@ -0,0 +1 @@
+// empty for now
```

Comment:
Well, otherwise it's a 404. This is until we actually add any code for service workers.